### ISSUE TITLE
Insufficient Color Contrast

### ISSUE THREAD

**Issue by jpdevries** [iancover/QuizApp#3]
Try installing the axe-core Chrome extension and auditing the quiz in the developer tools, it will point out 6 cases where color contrast is insufficient for accessibility requirements. For example

![](http://j4p.us/2M2x2q1c2e3V/Screen%20Shot%202017-06-13%20at%2012.59.33%20PM.png)

https://dequeuniversity.com/rules/axe/2.1/color-contrast?application=axeChrome
